src/NuGet.Licenses/App_Data/Licenses/PCRE2-exception.json

```diff
@@ -0,0 +1,7 @@
+﻿{
```

Comment:
Was the license already supported by us? Otherwise, we need to triage the case by case. Not sure what this license for? Same applies for others.

Reply:
I don't think this is a matter of licenses being supported or not. This repository keeps and maintain replicas of the licences and exceptions hold in https://spdx.org/licenses/  and https://spdx.org/licenses/exceptions-index.html. The script [license-text-extractor.ps1](https://github.com/NuGet/NuGet.Licenses/blob/main/src/NuGet.Licenses/license-text-extractor.ps1) in this repository does all the cloning and updating work. It was only a matter of chance (bug) that this script was stuck on downloading the exceptions from an old commit in the SPDX GitHub repository. Once I fixed the script, all newer license exception files were downloaded and processed automatically.

Same applies to all new files added in this patch. Actually, you can replicate this fix quite easily: just fix the script [license-text-extractor.ps1](https://github.com/NuGet/NuGet.Licenses/blob/main/src/NuGet.Licenses/license-text-extractor.ps1) as I did and run it.

Reply:
Let's scope to `Universal-FOSS-exception-1.0 license`. We need approval for new licenses on a case-by-case basis. Let's solve one problem at the time, there're not simple documents, actual legal document, we need to be careful.

Reply:
Some license supported by https://spdx.org/licenses/ doesn't mean we should immediately accept it.